zenml/Exposed_UI/README.md

```diff
@@ -0,0 +1,51 @@
+
+# common steps for both safe and unsafe zenml instances
+on an ubuntu 22.04 instance:
+```bash
+mkdir zenml
+cd zenml
+sudo add-apt-repository ppa:deadsnakes/ppa
+sudo apt update
+sudo apt install python3.11 python3.11-pip python3.11-venv
+python3.11 -m venv .venv
+source .venv/bin/activate
+# Install ZenML
+pip install "zenml[server]"
+# Start the ZenML dashboard locally.
+zenml up
```

Comment:
It would be better to add the `--ip-address` flag. This allows to bind zenml to an arbitrary address, otherwise this would only be exposed to localhost. If a user will deploy the testbed on an internet facing VM, this will not be reachable with the actual configuration proposed. 
A user could change the desired ip address at will.